styles/globals.css

```diff
@@ -1,6 +1,11 @@
 @import 'normalize.css';
 @import '../node_modules/bootstrap/dist/css/bootstrap-grid.min.css';
 
+:root {
+    --main-page-container-width: 350px;
+    --offset: 16%;
```

Comment:
Why `--main-page-container-width` is too short?

Is this the width of the sidebar?

Reply:
yes, I will rename it.